tests/infrastructure/golden_images/update_boot_source/test_ssp_data_sources.py

```
@@ -42,38 +42,61 @@ def opt_in_status_str(opt_in):
     return f"opt-{'in' if opt_in else 'out'}"
 
 
-def wait_for_data_source_reconciliation_after_update(
-    data_source, opt_in, volume_name_before_reconcile=DUMMY_VOLUME_NAME
+def wait_for_data_source_updated_referenced_volume(
+    data_source,
+    check_exact_match: bool,
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)